src/ping-thread.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (robot) ->
+	robot.hear /@(thread|t)\b/i, (context) ->
+		if context.robot.adapter.fetchThreadParticipants?
+			context.robot.adapter.fetchThreadParticipants context, (list) ->
+				list = list.filter (item) -> item not in ['@thread', '@t']
```

Comment:
This filter is necessary because the fetchThreadParticipants function returns not only the accounts that talked in the thread, but also those who were @'ed.  It does this in a mildly dumb fashion, not first checking to see if they're accounts, in order to take into account things like @team.  Since @thread also matches this criteria, and it is within the scope of ping-thread to be aware of this keyword, this is the place that filters them back out.